### DISCUSSION TITLE
Pipeline success/failure status icon color is troublesome for people with color vision deficiency

### DISCUSSION THREAD

**Discussion by gioravered-binahai** [community/community#46713] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Hi guys,
For people with color vision deficiency, aka color blindness (like me) - it is hard to differentiate the colors of the icons along the workflows (as seen in the image).
The X/V icon help a lot, but it would be great to have something that is more eye catching.
The Green tone and Red tone look almost similar to me.

<img width="142" alt="image" src="https://user-images.githubusercontent.com/11785178/217474122-3c64c020-bf21-4b1f-8cf7-6831116870b2.png">

Thank you very much,
Giora.

**Comment by ghostinhershell** [comment]
Hi @gioravered, 

Thanks so much for reaching out! I'm going to pull in my colleague, @benjiallen, here so he can take a closer look.

**Comment by cehfisher** [comment]
Hello @gioravered - thank you for providing this feedback! Have you tried using any of the [six GitHub themes](https://github.com/settings/appearance) aimed at supporting people with color deficiencies yet? If not, it might be something to check-out. If you have tried them out and believe further modifications are needed, please let us know! (cc @queenofcorgis)
